app/views/declaration/summary/additional_documents.scala.html

```diff
@@ -77,7 +77,7 @@
     else Some(SummaryListRow(
         classes = s"additional-documents-${item.sequenceId}-row",
         key = Key(content = Text(messages("declaration.summary.items.item.additionalDocuments"))),
-        value = Value(content = Text(messages("declaration.summary.items.item.additionalDocuments.none"))),
```

Comment:
Can these deprecated message keys be removed from the messages file?

Reply:
Did not I already remove them?

Reply:
Yes you did. My bad :+1: